public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="pl">
   <head>
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
It will throw the error because you deleted favicon.ico. So you should also remove this file. Better you could provide new favicon but with linkedin logo.

Reply:
I added a new linkedin logo icon.